app/controllers/thredded/topics_controller.rb

```diff
@@ -3,7 +3,7 @@ module Thredded
   class TopicsController < Thredded::ApplicationController
     before_action :thredded_require_login!,
                   only: [:edit, :new, :update, :create, :destroy]
-    before_action :update_user_activity
+    after_action :update_user_activity
```

Comment:
It just improves performance a bit, because currently the `:async` ActiveJob adapter runs the jobs in the same process (in a different thread, but same process, so it blocks on MRI).